test/wallet/index.ts

```diff
@@ -117,12 +117,9 @@ describe("Wallet", function () {
         SigningPubKey: publicKey,
       };
       const wallet = new Wallet(publicKey, privateKey);
-      const signedTx: { signedTransaction: string; id: string } =
-        wallet.signTransaction(txJSON);
+      const signedTx: string = wallet.signTransaction(txJSON);
 
-      assert.hasAllKeys(signedTx, ["id", "signedTransaction"]);
-      assert.isString(signedTx.id);
-      assert.isString(signedTx.signedTransaction);
+      assert.isString(signedTx);
```

Comment:
Signing is deterministic. So we might want to add an `expected` that was calculated directly via ripple-binary-codec.

Reply:
Do you have a specific function in mind?

Reply:
Basically check your work by doing what happens under the hood (by hand), just like you did with the multisign tests.